src/Uno.UWP/Storage/Streams/DataWriterStoreOperation.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Foundation;
+
+namespace Windows.Storage.Streams
+{
+	public partial class DataWriterStoreOperation : IAsyncOperation<uint>, IAsyncInfo
+	{
+		private readonly CancellationTokenSource _cts = new CancellationTokenSource();
+		private AsyncOperationCompletedHandler<uint> _onCompleted;
+		private AsyncStatus _status;
+		public DataWriterStoreOperation()
+		{
+			Status = AsyncStatus.Started;
+		}
+
+		public AsyncOperationCompletedHandler<uint> Completed
+		{
+			get { return _onCompleted; }
+			set
+			{
+				_onCompleted = value;
+				if (Status != AsyncStatus.Started) { value?.Invoke(this, Status); }
+			}
+		}
+
+		public Exception ErrorCode { get; set; }
+
+		public uint Id { get; set; }
+
+		public AsyncStatus Status
+		{
+			get => _status;
+			set => _status = value;
+		}
+
+		public void Cancel() => _cts.Cancel();
+
+		public void Close()
+		{
+			_cts.Cancel();
+			_cts.Dispose();
+		}
+
+		public uint GetResults()
+		{
+			return Status == AsyncStatus.Completed ? 1U : 0;
```

Comment:
According to the documentation, this should return the number of bytes in the operation, not `1U`.

https://docs.microsoft.com/en-us/uwp/api/windows.storage.streams.datawriterstoreoperation.getresults#returns

Reply:
Done. Thans for feedback.